File: exercicios_java/exercicios_java/src/secao7/exercicio3.java
//Programa que gera e escreve números ímpares dos números entre 100 e 200.

package secao7;

public class exercicio3 {
    public static void main(String[] args){
        for (int i=100; i <= 200; i++){
            if (i%2 != 0){
                System.out.println(i);
            }
        }
    }
}

File: exercicios_c/secao7/exercicio3.c
//- Programa que gera e escreve números ímpares dos números entre 100 e 200.

#include <stdio.h>

int main(){
    int a = 100;

    while (a <= 200){
        if (a%2 != 0){
            printf("%d\n", a);
        };
        a += 1;
    }
}